styles/camunda/docsInstance/hrefDocsToDocs.yml

```diff
@@ -0,0 +1,7 @@
+extends: existence
+message: "Improper link format: '%s'. Please specify the file extension."
+level: warning
+nonword: true
+scope: raw
+tokens:
+  - "\\[[^\\]]*\\]\\((\\.)?(\\/)?docs(?!.*(\\.md|\\.mdx))[^\\)]*\\)"
```

Comment:
This regex (and the others below) were tested by me with a variety of values. Trust me..????

Reply:
<img width="601" alt="image" src="https://github.com/camunda/camunda-docs/assets/1051363/3df83f43-43b3-496d-8733-6451b2567733">

I ran this through several regex testing tools, and they all barfed on the same forward slash. When I add another backslash, it goes away. Two should be enough to escape the forward slash, and it works... so maybe these tools are using the wrong engine?

Reply:
It's actually a reverse problem -- vale/go needs the backslashes double-escaped, so the testable regex would have only one backslash everywhere there are 2.

Super annoying, I agree. 

Would a note above each regex that it can be tested in a browser by removing one `\` in every pair of `\` be helpful?

I also considered adding unit tests of the regexes to the codebase -- but thought against it, because the configured rules wouldn't actually use the expression in the tests. (In two ways -- (a) the tested regex would have half the `\` characters as the rule, and (b) the rule wouldn't import the tested regex directly.)

Reply:
🤯 please add a note. I agree with your thoughts on the unit tests. I wouldn't add them. Seems ripe for confusion and chaos.